src/components/UnauthorizedApp.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import React, { Fragment } from 'react';
+import { Route, Switch } from 'react-router';
+import LoginForm from './auth/LoginForm';
+import RegisterForm from './auth/RegisterForm';
+import PasswordRequestForm from './auth/forgot_password/PasswordRequestForm';
+import ResetPasswordForm from './auth/forgot_password/ResetPasswordForm';
+import LandingPage from './landing_page/LandingPage';
+import UnauthorizedFooter from './footer/UnauthorizedFooter';
+import AuthorizedApp from './AuthorizedApp';
+import AuthenticatedRoute from './custom_routes/AuthenticatedRoute';
+import './UnauthorizedApp.scss';
+
+function UnauthorizedApp() {
+  return (
+    <Fragment>
+      <div className="UnauthorizedApp">
+        <Switch>
+          <Route path="/login" component={LoginForm} />
+          <Route path="/register" component={RegisterForm} />
+          <Route path="/forgot-password" component={PasswordRequestForm} />
+          <Route path="/reset-password" component={ResetPasswordForm} />
+          <Route exact path="/" component={LandingPage} />
+          <AuthenticatedRoute path="*" component={AuthorizedApp} />
```

Comment:
If more than one of these switch statements is true, where does it lead? Eg. if I am trying to visit /login while I am logged in, will the `          <Route path="/login" component={LoginForm} />
` or the `          <AuthenticatedRoute path="*" component={AuthorizedApp} />
` rule take precedence?